src/js/views/patients/patient/sidebar/sidebar_views.js

```diff
@@ -1,142 +1,141 @@
 import _ from 'underscore';
 import anime from 'animejs';
 import moment from 'moment';
+import Radio from 'backbone.radio';
 
 import hbs from 'handlebars-inline-precompile';
-import { View, CollectionView, Region } from 'marionette';
+import { View, CollectionView } from 'marionette';
 
-import intl, { renderTemplate } from 'js/i18n';
 import ModelRender from 'js/behaviors/model-render';
 
-import PatientSidebarTemplate from './patient-sidebar.hbs';
-import PatientSidebarGroupsTemplate from './patient-sidebar-groups.hbs';
-
 import './patient-sidebar.scss';
 import 'sass/domain/engagement-status.scss';
 
-const i18n = intl.patients.patient.sidebar.sidebarViews;
-
-const patientSet = [
-  {
-    heading: i18n.dobLabel,
+const sidebarWidgets = {
+  dob: {
     template: hbs`{{formatHTMLMessage (intlGet "patients.patient.sidebar.sidebarViews.dob") dob=(formatMoment dob "LONG" inputFormat="YYYY-MM-DD") age=age}}`,
-    data(patient) {
-      const dob = patient.get('birth_date');
+    templateContext() {
+      const dob = this.model.get('birth_date');
 
       const age = moment().diff(moment(dob, 'YYYY-MM-DD'), 'years');
 
       return { dob, age };
     },
   },
-  {
-    heading: i18n.sexLabel,
+  sex: {
     template: hbs`{{formatMessage (intlGet "patients.patient.sidebar.sidebarViews.sex") sex=sex}}`,
   },
-];
+  divider: {
+    template: hbs`<div class="patient-sidebar__divider"></div>`,
+  },
+  groups: {
+    behaviors: [
+      {
+        behaviorClass: ModelRender,
+        changeAttributes: ['_groups'],
+      },
+    ],
+    template: hbs`{{#each groups}}<div>{{ this.name }}</div>{{/each}}`,
+    templateContext() {
+      return {
+        groups: _.map(this.model.getGroups().models, 'attributes'),
+      };
+    },
+  },
+  engagement: View.extend({
+    modelEvents: {
+      'change:engagement': 'render',
+      'status:notAvailable': 'onStatusNotAvailable',
+    },
+    getTemplate() {
```

Comment:
I think this could be
```js
if (this.notAvailable) return hbs...
if (!this.model.get('engagment')) return hbs...
return hbs...
```
Also won't need templateContext that way